docs/BreakingChanges/147 BinaryFormatter can fail to find type from LoadFrom context.md

```diff
@@ -0,0 +1,44 @@
+## 147: BinaryFormatter can fail to find type from LoadFrom context
+
+### Scope
+Edge
+
+### Version Introduced
+4.5
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+As of .NET Framework 4.5, a number of XmlSerializer changes may cause differences in deserialization when using `BinaryFormatter` to deserialize types that had been loaded in the
```

Comment:
What's the connection between XmlSerializer and the BinaryFormatter? I thought they were separate serialization technologies.